code/index.html

```diff
@@ -16,6 +18,84 @@
       <img src="example-recipe-background.png" alt="Banana Bread">
 
     -->
-    <h1>Hello there! 👋🏼</h1>
+    <!--HEADER-->
+    <!-- <img src="pic1.jpg" alt="portrait" width="100%">   -->
+    <header>
+      <div class="header-text">
+        <h1>Sandy Vigo</h1>
+        <h2>Creating my world..</h2>
+      </div>  
+    </header>
+    <!--NAVIGATOR-->
+    <nav>
+      <a class="nav-link" href="#">About Myself</a>
+      <a class="nav-link" href="#">Healthy Food</a>
+      <a class="nav-link" href="#">Daily Exercises</a>
+      <a class="nav-link" href="#">Explore The World </a>   
+    </nav>
+    <!--CONTENT-->
+    <articel class="intro">
+      <div class="intro-picture">
+        <img src="me.jpg" width="20%" type=".jpg">
+      </div>
+      <div class="content-intro">
+        <p>I'm passionated about healthy food, cooking and tasting new dishes brings me joy. In combination with experimenting different types of sports and exercises, it gives me the perfect balance for a fulfilling everyday life.</p>
+        <br>
```

Comment:
To be more semantic in your HTML you could create the same space with margin or padding on one of the elements and avoid the nonsemantic tag `<br>`